regression-test/suites/schema_change_p0/test_add_keys_light_schema_change.groovy

```diff
@@ -0,0 +1,454 @@
+// under the License.
+
+import groovy.json.JsonOutput
+import org.codehaus.groovy.runtime.IOGroovyMethods
+
+suite ("test_add_keys_light_schema_change") {
+    def getJobState = { tableName ->
+         def jobStateResult = sql """  SHOW ALTER TABLE COLUMN WHERE IndexName='${tableName}' ORDER BY createtime DESC LIMIT 1 """
+         return jobStateResult[0][9]
+    }
+
+    def getJobTxnId = { tableName ->
+         def jobStateResult = sql """  SHOW ALTER TABLE COLUMN WHERE IndexName='${tableName}' ORDER BY createtime DESC LIMIT 1 """
+         return jobStateResult[0][8]
+    }
+
+    def getCreateViewState = { tableName ->
+        def createViewStateResult = sql """ SHOW ALTER TABLE MATERIALIZED VIEW WHERE IndexName='${tableName}' ORDER BY createtime DESC LIMIT 1 """
+        return createViewStateResult[0][8]
+    }
+
+    def waitJobFinish = { strSql, statePos -> 
+        Object jobStateResult = null
+        int max_try_time = 2000
+        while (max_try_time--){
+            jobStateResult = sql """${strSql}"""
+            def jsonRes = JsonOutput.toJson(jobStateResult)
+            String result = jobStateResult[0][statePos]
+            if (result == "FINISHED") {
+                log.info(jsonRes)
+                sleep(500)
+                break
+            } else {
+                sleep(100)
+                if (max_try_time < 1){
+                    assertEquals(1,2)
+                }
+            }
+        }
+        return jobStateResult
+    }
+    
+    def tableName = "add_keys_light_schema_change"
+    try {
+
+        String backend_id;
+        def backendId_to_backendIP = [:]
+        def backendId_to_backendHttpPort = [:]
+        getBackendIpHttpPort(backendId_to_backendIP, backendId_to_backendHttpPort);
+
+        backend_id = backendId_to_backendIP.keySet()[0]
+        def (code, out, err) = show_be_config(backendId_to_backendIP.get(backend_id), backendId_to_backendHttpPort.get(backend_id))
+        
+        logger.info("Show config: code=" + code + ", out=" + out + ", err=" + err)
+        assertEquals(code, 0)
+        def configList = parseJson(out.trim())
+        assert configList instanceof List
+
+        boolean disableAutoCompaction = true
+        for (Object ele in (List) configList) {
+            assert ele instanceof List<String>
+            if (((List<String>) ele)[0] == "disable_auto_compaction") {
+                disableAutoCompaction = Boolean.parseBoolean(((List<String>) ele)[2])
+            }
+        }
+        int max_try_time = 3000
+        List<List<Object>> table_tablets = null;
+        List<List<Object>> rowset_cnt = null;
+
+        // case 1.1: light schema change type check: add key column in short key column num
+        sql """ DROP TABLE IF EXISTS add_keys_light_schema_change """
+        sql """
+                CREATE TABLE IF NOT EXISTS add_keys_light_schema_change (
+                    `user_id` LARGEINT NOT NULL COMMENT "用户id",
```

Comment:
should test all typed key columns such as string,date,int.
For example, the original key column could be one of string,date,int 
and you add a new key column, it could also be one of string,date,int